### ISSUE TITLE
JBrowse desktop (occasionally?) crashes on quit on mac

### ISSUE THREAD

**Issue by cmdcolin** [GMOD/jbrowse-components#4222]
on selecting "Quit" from the jbrowse desktop mac app, i sometime see a crash.

tested with v2.10.2

not sure if it's just my machine or not, and it seems like it is not be every time...

<img width="1002" alt="Screenshot 2024-02-19 at 5 20 13 PM" src="https://github.com/GMOD/jbrowse-components/assets/6511937/81114d29-b9da-4208-b700-ad84faa63d3d">

**Comment by cmdcolin** [comment]
@carolinebridge-oicr @scottcain are you able to confirm this on mac at all? occaisionally jbrowse desktop quitting crashes?

**Comment by scottcain** [comment]
I haven't seen it when I use Desktop, but I've only used 2.10.2 a few times.

**Comment by carolinebridge-oicr** [comment]
Played around with this a bit with sessions of various sizes and runtimes -- could not reproduce. Were you running a specific plugin?

**Comment by cmdcolin** [comment]
No, it was quitting directly from the start screen right after starting it, no session loading, no plugins. It's a newer mac (ya... i got a mac) and intel based. Its a pretty weird error but if you aren't seeing it locally then it may just be me

**Comment by carolinebridge-oicr** [comment]
Reproduced it -- just by opening and quitting from the start screen

![image](https://github.com/GMOD/jbrowse-components/assets/83305007/da62eabb-5ff1-41d5-917d-5ac8234a4482)

**Comment by scottcain** [comment]
Hmm. I couldn't get it to do that on my M1 Mac. (OS 14.2.1)

**Comment by cmdcolin** [comment]
probably can close...not seeing
